src/clashboard/clinical_trials_data.py

```diff
@@ -7,37 +7,36 @@ def __init__(self):
         self.type_counts = None
         self.curr_group = ''
         self.studies = pd.DataFrame()
+        self.filters = []
 
-    # Uncomment when tested and implemented
-    # def remove_filter(self, filter_category, filter_name):
-    #    """
-    #    Removes a specified filter from the list of filters,
-    #       and re-runs the query
-    #    :param filter_category:
-    #           the column in the DB that the user wants to filter
-    #    :param filter_name:
-    #           the specific value to filter on in that column
-    #    """
-    #    pass
+    def remove_filter(self, filter_category, filter_name):
+        """
+        Removes a specified filter from the list of filters,
+           and re-runs the query
+        :param filter_category:
+               the column in the DB that the user wants to filter
+        :param filter_name:
+               the specific value to filter on in that column
+        """
+        self.filters.remove([filter_category, filter_name])
```

Comment:
Will we need any sort of validation for these before we assume they can remove a filter? 
This can probably be solved through more extensive testing. 
Or, maybe validation here might be too defensive... Probably a place of discussion.